package.json

```
@@ -33,12 +33,16 @@
 		"Jordan Smith <jordan@taskrabbit.com>",
 		"Michael Kachanovskyi <mikach27@gmail.com>",
 		"Robert Murray <rmurray@taskrabbit.com>",
-		"Zane Chua <github.com/zanechua>"
+		"Zane Chua <github.com/zanechua>",
+		"Eric Adamski <github.com/ericadamski>"
 	],
 	"license": "MIT",
-	"devDependencies": { "prettier": "2.0.x", "eslint": "7.3.x" },
+	"devDependencies": {
+		"eslint": "7.3.x",
+		"prettier": "2.0.x"
+	},
 	"peerDependencies": {
 		"react": "^16.11.0",
-		"react-native": "0.60.x | 0.61.x | 0.62.x | 0.63.x"
+		"react-native": "*"
```

Not sure why this changed either. Reverting.

```suggestion
		"react-native": "0.60.x | 0.61.x | 0.62.x | 0.63.x"
```